tensorflow/core/kernels/data/dataset_test_base.cc

```diff
@@ -293,22 +292,61 @@ Status DatasetOpsTestBase::MakeRangeDataset(
   return Status::OK();
 }
 
-// Create a `RangeDataset` dataset as a variant tensor.
 Status DatasetOpsTestBase::MakeRangeDataset(
```

Comment:
similar to my comment about dataset_params, this should go into `range_dataset_op_test.h` ... putting all test dataset into this files does not scale

Reply:
Yeah, I'm also planning to change this. After `DatasetParams` is updated with `MakeInputs()` and `MakeAttributes()`, we do not need to create a similar function for each dataset. A more general function can be used to replace them as below. 
```C++
Status DatasetOpsTestBase::MakeDatasetTensor(
      DatasetParams& dataset_params, 
      const FunctionDef& make_deataset_func, 
      Tensor* dataset_tensor);
```